src/components/OrderPage/index.js

```diff
@@ -1,9 +1,52 @@
 import {useEffect, useState} from "react"
 import ItemCard from "../ItemCard"
 import './OrderPage.scss'
+import OrderCard from "./OrderCard";
+import userEvent from "@testing-library/user-event";
 
 const OrderPage = ({activeRestaurantId}) => {
     const [foodItemData, setItemData] = useState([])
+    const [orderObject, setOrderObject] = useState([])
+
+
+    const addItem = (foodName, price) => {
+        let orderObjectClone = [...orderObject]
+        const orderObjectItem = {foodName : foodName, price : price, qty: 1}
```

Comment:
There's a small improvement you can make here.

If you're setting an item in an object to be a variable with the same key as the variable name (like `foodName: foodName`), you can use a shorthand like this:

```
{
    foodName,
    price,
    qty: 1
}
```

This will automatically create keys for the variables with the same name. Let me know if this doesn't make sense